_drafts/2017-10-13-rprofile-david-smith.md

```diff
@@ -0,0 +1,118 @@
+---
+name: rprofile-david-smith
+layout: post_discourse
+title: ".rprofile: David Smith"
+date: 2017-10-13
+authors:
+  - name: Kelly O'Briant
+    url: https://kellobri.github.io/
+tags:
+- R
+- community
+- interviews
+---
+
+<div class="alert alert-info" role="alert">
+In this occasional series, we interview someone using a loosely defined set of interview questions for the purpose of “Demystifying the Creative/Development Process of the R Community”. This interview was conducted and prepared by Kelly O'Briant as part of an rOpenSci unconf17 project.
+</div>
+
+<br>
+
+<img src="/assets/blog-images/2017-10-13-rprofile_david_smith/david-smith.jpg" alt="David Smith, R Community Lead at Microsoft" align="left" style="margin: 0px 20px", width="250px">
```

Comment:
@sckott does dir name need to have hyphens here?

Reply:
image shows up when rendered on my machine, looks good